test/no-validate-leaksan.txt

```diff
@@ -0,0 +1,3291 @@
+# List of tests for which we do NOT enable LeakSanitizer when running in ASAN CI
```

Comment:
This list should be inverted until a reasonable number of leak tests are fixed because anyone adding a new test will need to add it to this file right now

Instead it should be empty and passing tests can be added to this file. Then once most tests pass, it can be flipped.

Reply:
i think this was a good identification but itd be better for this to not merge yet https://github.com/oven-sh/bun/pull/21142#issuecomment-3099721799